Tasks/Salanenka/Task2/Task2.py

```diff
@@ -0,0 +1,19 @@
+dict = {9103976271:[("Reina", "Meinhard"), ("Memphis", "Tennessee")], 4199392609:[("Stephanie", "Bruce"), ("Greensboro", "North Carolina")],
+9099459979:[("Ermes", "Angela"), ("Dallas", "Texas")], 6123479367:[("Lorenza", "Takuya"), ("Indianapolis", "Indiana")], 7548993768:[("Margarete", "Quintin"), ("Raleigh", "North Carolina")]}
+while True: 
+    input_number = input ("Enter the phone number in 1234567890 format: \n")
+    if (input_number.isdigit()) == False or len(input_number) != 10:
```

Comment:
**(input_number.isdigit()) == False** - this operation is full of redundancy, first of all it would work without parentheses **input_number.isdigit() == False** , secondly it's being used in a logical expression means that the comparison to **False** is not needed as well
it should be like  **if len(input_number) != 10 or not input_number.isdigit()**

Reply:
Thanks! will study that